src/NodaTime.Demo/OffsetDemo.cs

```diff
@@ -62,6 +62,19 @@ public void ConstructionFromTimeSpan()
             Assert.AreEqual(5400, offset.Seconds);
         }
 
+        [Test]
+        public void Max()
+        {
+            var offset = Offset.FromSeconds(10);
```

Comment:
I would call these two "smaller" and "larger", probably - and use more common offsets, e.g. one hour and three hours.